mne_bids/read.py

```diff
@@ -220,12 +220,10 @@ def _float_or_nan(val):
         if any(np.isnan(ch_coord)) and ch_name not in raw.info['bads']:
             nan_chs.append(ch_name)
     if len(nan_chs) > 0:
-        logger.warning("There are problematic channels "
-                       "that are not set as 'bad', "
-                       "but also the channel location "
-                       "is 'n/a' in electrodes.tsv: {}. "
-                       "Setting montage without "
-                       "those chs...".format(nan_chs))
+        warn("There are problematic channels that are not "
+             "set as 'bad', but also the channel location "
+             "is 'n/a' in electrodes.tsv: {}. Setting "
+             "montage without those chs...".format(nan_chs))
```

Comment:
Let me suggest something like this

`"There are channels without locations (n/a) that are not marked as bad: {}".format(nan_chs)`

ok for you?

Reply:
fine by me.